### PR TITLE
New look with Bootstrap

### PR DESCRIPTION
**Pull request by echofoxtrotpl**
- [ ] Check the live version of website here https://gods-eye-project.herokuapp.com/
- [ ] Try to login with your name and again your name, both lowercase
- [ ] Look into the *.ejs files in view folder and tell me if something is wrong
- [ ] Check how does the website display on different devices
- [ ] Leave your comments :smile:

### CODE REVIEW

**Comment by echofoxtrotpl** [comment]
Screenshot_2020-11-19-07-47-43-210_com android chrome
One of the problems i don't know how to slove 😞

**Comment by Tom2rec** [comment]
I think that new look is great :clap: and I don't have any problems with displaying this page - testing on safari mobile and firefox desktop.

**Comment by sokoloowski** [comment]
> Screenshot_2020-11-19-07-47-43-210_com android chrome
> One of the problems i don't know how to slove

This pop-up is connected with browser. Google is checking given password with leaked passwords database. I think there was leak from Heroku in the past and one of passwords was like Yours

**Comment by sokoloowski** [comment]
That look is great! Love it :heart_eyes:

**Review by Tom2rec — COMMENTED** [review]
I didn't find any suspicious section :+1:

File: Web/views/index.ejs [hunk 1]
```diff
@@ -2,11 +2,93 @@
...
+              <li class="nav-item">
+                <a class="nav-link active" href="/">
+                  <span data-feather="home"></span>
+                  Dashboard <span class="sr-only">(current)</span>
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="/hash">
+                  <span data-feather="users"></span>
+                  Account
```
Comment (sokoloowski) [inline Web/views/index.ejs:35]: I recommend changing it to "Change password" or creating user info under that link

File: Web/views/index.ejs [hunk 2]
```diff
@@ -2,11 +2,93 @@
...
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">
+                  <span data-feather="server" target="_blank"></span>
+                  Server
```
Comment (sokoloowski) [inline Web/views/index.ejs:41]: Unnesessary, as main page is available under link at the very top of sidebar

File: Web/views/index.ejs [hunk 3]
```diff
@@ -2,11 +2,93 @@
...
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="/hash">
+                  <span data-feather="users"></span>
+                  Account
+                </a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">
+                  <span data-feather="server" target="_blank"></span>
```
Comment (sokoloowski) [inline Web/views/index.ejs:40]: `<span>` does not take `target` attribute
```suggestion
                  <span data-feather="server"></span>
```

File: Web/views/index.ejs [hunk 4]
```diff
@@ -2,11 +2,93 @@
...
+                  Slack
+                </a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+    
+        <main role="main" class="col-md-9 ml-sm-auto col-lg-10 px-md-4">
+          <div class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pt-3 pb-2 mb-3 border-bottom">
+            <h1 class="h2">Dashboard</h1>
```
Comment (sokoloowski) [inline Web/views/index.ejs:74]: Just use `<h2>` here!
```suggestion
            <h2>Dashboard</h2>
```

File: Web/views/login.ejs [hunk 5]
```diff
@@ -1,16 +1,23 @@
...
-    <div class="login_form"> 
-        <form>
-            <h2>Sign in :)</h2>
-            <label for="email"><input type="text" name="email" placeholder="email" required/></label>
-            <label for="password"><input type="password" name="password" placeholder="password" required/></label>
+<link href="/signin.css" rel="stylesheet">
+</head>
+<body class="text-center">  
+        <form class="form-signin">
+            <h1 class="h2 mb-3 font-weight-normal">Please sign in</h1>
```
Comment (sokoloowski) [inline Web/views/login.ejs:6]: Just use `<h2>`
```suggestion
            <h2 class="mb-3 font-weight-normal">Please sign in</h2>
```

File: Web/views/partials/head.ejs [hunk 6]
```diff
@@ -4,8 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <!-- to import css, add it to public folder and add: app.use(express.static('public')) to app.js -->
-    <link rel="stylesheet" href="/styles.css"> 
-    <!-- import tile form acutal request -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
     <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <!-- import tile form acutal request -->
```
Comment (sokoloowski) [inline Web/views/partials/head.ejs:9]: Misspelled "title" in comment
```suggestion
    <!-- import title form acutal request -->
```

File: Web/views/partials/head.ejs [hunk 7]
```diff
@@ -4,8 +4,7 @@
...
     <title>God'sEye | <%= title %></title>
```
Comment (sokoloowski) [inline Web/views/partials/head.ejs:10]: Missing space
```suggestion
    <title>God's Eye | <%= title %></title> 
```

**Review by sokoloowski — COMMENTED** [review]
Minor changes required in HTML structure

**Comment by sokoloowski** [comment]
image
Audit performed via LightHouse in Google Chrome

File: Web/views/partials/head.ejs [hunk 8]
```diff
@@ -4,8 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (sokoloowski) [inline Web/views/partials/head.ejs:5]: Document does not have a meta description
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="God sees everything, doesn't He?">
```

**Review by sokoloowski — CHANGES_REQUESTED** [review]
Some changes are really important from SEO point of view:
image
Please, fix your files before merge

**Comment by echofoxtrotpl** [comment]
Thank you for your advices, I am getting down to work ;)

**Comment by echofoxtrotpl** [comment]
Everything done, besides SEO optimization. I think that we should do it later because we only need simple website until we have data on it.

**Review by sokoloowski — APPROVED** [review]
Everything seems to be good!

**Pull request merged**